internal/auth/auth.go

```diff
@@ -9,60 +9,58 @@ import (
 )
 
 var (
-	// UserToken is the retrieved user token from the token file.
-	// It must be set using ReadToken.
-	UserToken string
-
 	// TokenDir is the directory for the token file.
 	TokenDir = ".config/benchttp" // nolint:gosec // no creds
 	// TokenName is the name for the token file.
 	TokenName = "token.txt"
 
-	// ErrTokenFind reports an error finding a token file.
-	ErrTokenFind = errors.New("token not found")
+	// ErrTokenPath reports an error resolving a token path.
+	ErrTokenPath = errors.New("path error")
 	// ErrTokenRead reports an error reading a token file.
 	ErrTokenRead = errors.New("invalid token")
 	// ErrTokenSave reports an error saving a token file.
 	ErrTokenSave = errors.New("cannot save token")
 )
 
-// ReadToken reads a token from a file and sets UserToken to the retrieved
+// userToken is the retrieved user token from the token file.
+// It must be set using ReadToken.
+var userToken string
```

Comment:
`userToken` is never assigned a value. That or I missed it.

Either way, we should only read the token once, there's no need to cache it (that's what this declaration does + `if userToken != ""` statement.

It think we can delete lines 25-27 and 32-35 (and update the doc of `ReadToken`) and there's no behavior changes.

Correct me if I'm wrong

Reply:
Yep correct, the goal was to set `userToken` at the end of `ReadToken` and returns its value, that way the next call would only read its value if already set. But I forgot about it halfway through 😅 

Deleted the whole thing as we don't have a use case for it.